mentorship ios/ru.lproj/Localizable.strings

```diff
@@ -2,6 +2,6 @@
   Localizable.strings
   mentorship ios
 
-  Created by Yugantar Jain on 04/06/20.
+  Created on 04/06/20.
   Copyright © 2020 Yugantar Jain. All rights reserved.
 */
```

Comment:
Copyright line is still present.

Reply:
I think ru.lproj is left over from localization testing. These files aren't actually part of the project now. I'll remove them.